metricq/source.py

```diff
@@ -45,7 +46,7 @@
 
 logger = get_logger(__name__)
 
-MetadataDict = Dict[str, Any]
+MetadataDict = dict[str, Any]
```

Comment:
```suggestion
MetadataDict = JsonDict
```

Might as well replace it.

Reply:
Not remove. I'd rather toy around with TypedDict plus  typeddict-unknown-key